src/core/model_config.proto

```diff
@@ -661,6 +725,31 @@ message ModelDynamicBatching
   //@@     order.
   //@@
   bool preserve_ordering = 3;
+
+  //@@  .. cpp:var:: ModelQueuePolicy default_queue_policy
+  //@@
+  //@@     The default queue policy used for requests that don't require
+  //@@     priority handling and requests that specify priority levels where
+  //@@     there is no specific policy given. If not specified, a policy with
+  //@@     default field values will be used.
+  //@@
+  ModelQueuePolicy default_queue_policy = 4;
+
+  //@@  .. cpp:var:: uint32 number_of_priority_level
+  //@@
+  //@@     The number of priority levels to be enabled for the model,
```

Comment:
Any reason why we don't start priority level at 0?
You should make the highest priority be level 0 and higher priority values be lower priority.

Reply:
There is a priority level field in request header, and the default value is 0 if user doesn't specify a priority level. Given that, 0 is reserved for "no priority".

Reply:
I see. If the request doesn't specify a priority what queue are you going to put it in? You have a default_queue_policy but you still need to know what queue to use. Seems like you also need a default_priority_level.

Also for priority_levels should also have a value of 0 (the default) mean that there is no prioritization at all... I think that it is equivalent to priority_levels == 1.